### PR TITLE
P21 Cruise-specific Cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Profiler Mooring ADCP cal csvs
Surface Mooring METLGR cal csvs

References:
mooring configs in Pioneer 21 folder in Vault
METBK_Module_Heights.xlsx in Sensors Notebook/METBK in Vault (same as NES)
3408-00201 cal csv procedure

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Pioneer 21 AM Updates](https://trello.com/c/65WTFp4J/252-pioneer-21-am-updates)

File: calibration/METBKA/CGINS-METLGR-00037__20241204.csv
Comment (reedan88) [inline calibration/METBKA/CGINS-METLGR-00037__20241204.csv:1]: Looks good.

File: calibration/METBKA/CGINS-METLGR-00017__20241204.csv
Comment (reedan88) [inline calibration/METBKA/CGINS-METLGR-00017__20241204.csv:1]: Looks good.

File: calibration/ADCPSL/CGINS-ADCPSL-18446__20241204.csv
Comment (reedan88) [inline calibration/ADCPSL/CGINS-ADCPSL-18446__20241204.csv:1]: Looks good.

File: calibration/ADCPSL/CGINS-ADCPSL-21393__20241204.csv
Comment (reedan88) [inline calibration/ADCPSL/CGINS-ADCPSL-21393__20241204.csv:1]: Looks good

File: calibration/ADCPTG/CGINS-ADCPTG-18660__20241204.csv
Comment (reedan88) [inline calibration/ADCPTG/CGINS-ADCPTG-18660__20241204.csv:1]: Looks good.

File: calibration/ADCPTG/CGINS-ADCPTG-20495__20241204.csv
Comment (reedan88) [inline calibration/ADCPTG/CGINS-ADCPTG-20495__20241204.csv:1]: Looks good.

File: calibration/ADCPTG/CGINS-ADCPTG-20499__20241204.csv
Comment (reedan88) [inline calibration/ADCPTG/CGINS-ADCPTG-20499__20241204.csv:1]: Looks good.

**Review by reedan88 — APPROVED** [review]
Looks good.

File: calibration/METBKA/CGINS-METLGR-00017__20241204.csv [hunk 1]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR017,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP10CNSM-00002 METBK1
+LGR017,CC_height_of_air_humidity_measurement_m,4.255,
+LGR017,CC_height_of_air_temperature_measurement_m,4.255,
+LGR017,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR017,CC_jcool,1,Constant
+LGR017,CC_jwarm,1,Constant
+LGR017,CC_zinvpbl,600,Constant
+LGR017,CC_use_velpt,1,Constant
```
Comment (cooleyky) [inline calibration/METBKA/CGINS-METLGR-00017__20241204.csv:9]: looks good

File: calibration/METBKA/CGINS-METLGR-00037__20241204.csv [hunk 2]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR037,CC_depth_of_conductivity_and_temperature_measurements_m,1.11,CP10CNSM-00002 METBK2
+LGR037,CC_height_of_air_humidity_measurement_m,4.255,
+LGR037,CC_height_of_air_temperature_measurement_m,4.255,
+LGR037,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR037,CC_jcool,1,Constant
+LGR037,CC_jwarm,1,Constant
+LGR037,CC_zinvpbl,600,Constant
+LGR037,CC_use_velpt,1,Constant
```
Comment (cooleyky) [inline calibration/METBKA/CGINS-METLGR-00037__20241204.csv:9]: looks good

**Review by cooleyky — APPROVED** [review]
Looks good

**Pull request merged**